samples/log_callback/log_callback.edl

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/syscall.edl" import *;
+    from "platform.edl" import *;
+
+    trusted {
+        public void enclave_log_callback();
```

Comment:
This ecall and the ocall below are themselves not logging callbacks. Therefore the names are misleading.
You could just call them: enc_hello and host_hello.

Reply:
Thanks for the suggestion, both are fixed.